src/main/webapp/data/userMapData.json

```diff
@@ -829,6 +847,7 @@
     ],
     "Sweden": [
       "International School of the Stockholm Region",
+      "Jönköping International Business School",
```

Comment:
This is actually part of `Jönköping University` which is an existing entry one line below.